### ISSUE TITLE
html-webpack-plugin/lib/hooks is internal, breaks with 5.6.2

### ISSUE THREAD

**Issue by aleho** [drolsen/webpack-favicons#23]
https://github.com/drolsen/webpack-favicons/blob/f1d12eb313e03fb309f5c18498dd0178e99da0ca/index.js#L78

According to https://github.com/jantimon/html-webpack-plugin/issues/1866#issuecomment-2430566887 this should be fixed by using the correct module:

```js
require('html-webpack-plugin').getCompilationHooks(compilation).alterAssetTags.tapAsync(
```

There's a fix with 5.6.3, but the author suggests to update the code anyway.

**Comment by drolsen** [comment]
I've published a new 1.5.0 version.
Please give this a run, but know that it bumps the min node version up from 14.17.0 to 18.17.0

Thanks!

**Comment by aleho** [comment]
Thank you very much!

Just updated the project (to 1.5.4) and everything (I use) seems to work.
